File: nan21.dnet.module.md.presenter/src/main/java/net/nan21/dnet/module/md/presenter/impl/mm/model/ProductLov_Ds.java
/**
 * DNet eBusiness Suite
 * Copyright: 2010-2013 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms.
 */
package net.nan21.dnet.module.md.presenter.impl.mm.model;

import net.nan21.dnet.core.api.annotation.Ds;
import net.nan21.dnet.core.api.annotation.SortField;
import net.nan21.dnet.core.presenter.model.AbstractTypeWithCodeLov;
import net.nan21.dnet.module.md.domain.impl.mm.Product;

@Ds(entity = Product.class, sort = {@SortField(field = ProductLov_Ds.f_code)})
public class ProductLov_Ds extends AbstractTypeWithCodeLov<Product> {

	public ProductLov_Ds() {
		super();
	}

	public ProductLov_Ds(Product e) {
		super(e);
	}
}

File: nan21.dnet.module.md.ui.extjs/src/main/resources/webapp/js/ds/ProductLov_Ds.js
/**
 * DNet eBusiness Suite
 * Copyright: 2010-2013 Nan21 Electronics SRL. All rights reserved.
 * Use is subject to license terms.
 */
	
Ext.define(Dnet.ns.md + "ProductLov_Ds", {
	extend: 'Ext.data.Model',
	
	fields: [
		{name:"id", type:"string"},
		{name:"clientId", type:"string"},
		{name:"code", type:"string"},
		{name:"name", type:"string"},
		{name:"description", type:"string"},
		{name:"active", type:"boolean"},
		{name:"refid", type:"string"}
	]
});

Ext.define(Dnet.ns.md + "ProductLov_DsParam", {
	extend: 'Ext.data.Model',
	
	fields: [
		{name:"companyIdForNewProdAccount", type:"string"}
	]
});